helm-charts/example-apiserver/templates/konk-service.yaml

```diff
@@ -6,8 +6,8 @@ spec:
   konk:
     name: example-konk
   service:
-    name: runner-example-apiserver
-  group: example.infoblox.com
-  version: v1alpha1
+    name: {{ include "example-apiserver.fullname" . }}
+  group: {{ .Values.group }}
+  version: {{ .Values.version }}
```

Comment:
This will likely be static for any given app, so they don't really need to be values.

Reply:
I figured it would be easier to add these fields to values, since its also used here (forgot to override)- https://github.com/infobloxopen/konk/blob/main/helm-charts/example-apiserver/templates/apiservice.yaml#L18-L19

Although I don't believe we need to create this in the external cluster.